client-src/elements/form-field-specs.ts

```diff
@@ -109,9 +109,12 @@ export function resolveFieldForFeature(
  * Removing single quote ('), backtick (`), and pipe (|) since they are risky unless properly escaped everywhere.
  * Also removing ! and % because they have special meaning for some older email routing systems. */
 const USER_REGEX: string = String.raw`[A-Za-z0-9_#$&*+\/=?\{\}~^.\-]+`;
-const DOMAIN_REGEX: string = String.raw`(([A-Za-z0-9\-]+\.)+[A-Za-z]{2,6})`;
+const DOMAIN_NAME_REGEX: string = String.raw`(([A-Za-z0-9\-]+\.)+[A-Za-z]{2,6})`;
+const LOCALHOST_REGEX: string = String.raw`(localhost|127\.0\.0\.1)`;
+const DOMAIN_REGEX: string =
+  '(' + DOMAIN_NAME_REGEX + '|' + LOCALHOST_REGEX + ')';
 
-const EMAIL_ADDRESS_REGEX: string = USER_REGEX + '@' + DOMAIN_REGEX;
+const EMAIL_ADDRESS_REGEX: string = USER_REGEX + '@' + DOMAIN_NAME_REGEX;
```

Comment:
One question - how is this regex being used and how is it related to the LOCALHOST_REGEX?

Reply:
When developing locally, attachments are served from the local host rather than a regular domain name.   So, the URL validation regex that was looking for something like `google.com` was rejecting `127.0.0.1`.  I implemented it as a separate regex because we don't need that for email addresses.